docs/writing-pipelines.md

```diff
@@ -35,19 +35,18 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 nameSuffix: -go-pipeline
 resources:
-  - ../../../components/base/pipeline.yaml
-  - ../../../components/base/task.yaml
+  - /components/base/pipeline.yaml
```

Comment:
I don't think these resources are needed any more, they should be injected automatically by the execution engine

Reply:
Good spot